src/app/lib/button/button-demo/button-demo.component.html

```diff
@@ -19,7 +27,6 @@ <h5>
         Disabled Buttons
     </h5>
     <button hc-button title="Disabled Button" color="primary" [disabled]="true">Disabled Button</button>
-    <a hc-button title="Disabled Link" color="primary" [disabled]="true">Disabled Link</a>
     <h5>
         Tertiary Buttons
```

Comment:
It would be nice to have the anchor component near here to compare to the tertiary buttons

Reply:
Completely agree @benjanderson - I'll get that added today and double check the styling on that anchor component.  Then hopefully this pull request will be ready to be merged.